src/content/configuration/output.md

```diff
@@ -431,31 +430,17 @@ For details see [`output.jsonpFunction`](#outputjsonpfunction).
 
 ## `output.hotUpdateMainFilename`
 
-`string` `function`
-
-Customize the main hot update filename. See [`output.filename`](#outputfilename) option for details on the possible values.
+`string: '[hash].hot-update.json'` `function`
 
-`[hash]` is the only available placeholder, the default being:
-
-__webpack.config.js__
-
-```javascript
```

Comment:
Removed example, it was default. Doesn't look like the function is being used at all, maybe we need to flag this in issues: https://github.com/webpack/webpack/search?q=hotUpdateMainFilename&unscoped_q=hotUpdateMainFilename